paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -234,6 +236,12 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
             }
         }
 
+        viewModelScope.launch {
+            saveForFutureUse.onEach { saveForFutureUse ->
+                updateScreenStateWithSaveForFutureUse(saveForFutureUse)
```

Comment:
We need to update the screen state here to switch between `By continuing` and `By saving`.